lib/http/HttpRequestEncoder.cpp

```diff
@@ -45,6 +45,7 @@ namespace MAT_NS_BEGIN {
         ctx->httpRequest->SetUrl(m_config.GetCollectorUrl());
 
         ctx->httpRequest->GetHeaders().set("Expect", "100-continue");
+        ctx->httpRequest->GetHeaders().set("CV", ctx->cvStr);
```

Comment:
>set [](http://example.com/codeflow?start=39&length=3)

Can we make this assignment based upon some compiler define so we can configure it only for MIP?

Reply:
Do we need it. asking since we will be using it for MIP only. Or you mean for scenario, going forward we may put this change in global 1DS repo

Reply:
Made required changes

Reply:
> going forward we may put this change in global 1DS repo

I think adding this header would be wrong. This is not the tech we should be using going forward.

Reply:
It should be fine as long as these changes are confined to a separate branch, for debugging purpose, and not added to the `main` branch.